backend/blueprint/api/controllers/generateBadges.py

```diff
@@ -21,14 +20,15 @@ def generateBadges():
                 str(e),
                 'Could not find any JSON'))
 
-    if not data.get('csv'):
+    if not data['csv']:
```

Comment:
will this not raise exception if csv is not present?

Reply:
`.get` did not seem to work on my local. I had to make these changes to make it work.
So I changed it to this.